frontend/tools-page.html

```diff
@@ -90,478 +90,254 @@
         </div>
       </div>
     <!--  Tools  -->
-    <div class="section tools-page">
-        <div class="tools-simple-container tools-first-section">
-            <h2 class="tools-section-label">Online Presence Tools</h2> 
-            <div class="row "> 
-                <div class="col m12 l6 s12">
-                    <div class="tools-card tools-horizontal course-card">
-                        <div class="tools-card-image">
-                            <img class="icon-img" src="images/wix.jpg">
-                        </div>
-                        <div class="tools-card-stacked">
-                            <div class="tools-card-content">
-                                <h2>Wix</h2>
-                            </div>
-                            <div class="card-action">
-                                <div class="visit-link">
-                                <a href="#!">VISIT SITE</a>
-                                </div>
-                            </div>
-                        </div>
-                    </div>  
+    <div class="hr-sect">Google Tools</div>
+        <div class="row">
+            <div class="col l4 m6 s12">
+                <div class="card">
+                    <img class="card-img-top" src="images/google-search.jpg" alt="Card image cap">
+                    <div class="card-block">
+                        <h4 class="card-title">Google Search</h4>
+                        <p class="card-text">Research with precision. Simple tools and tips to help you get better results, faster.</p>
+                        <a href="#" class="card-link">VIEW LESSONS</a>
```

Comment:
@andela-iakande Is it right that frontend/tools-page is copied over to coursebuilder/modules/explorer/_static/components/pages/tools-page.html and therefore they should always be similar?

Reply:
@chegejames the frontend directory serves as the initial platform I often use to kickstart any development process before taking it to the coursebuilder app because it is faster and easier to manage in that manner. Apparently, we might not need it later on.
@andela-angene buttressed it here..  https://github.com/CodeForAfricaLabs/coursebuilder-core/pull/1.

Reply:
Ok. For now, let us keep that workflow, it reduces errors since you only have one copy to edit.

Reply:
ok. Thanks